src/libraries/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Specific.cs

```diff
@@ -25,6 +25,17 @@ public void InvalidConnectTimeout_Throws_ArgumentOutOfRangeException()
                 AssertExtensions.Throws<ArgumentOutOfRangeException>("timeout", () => { client.ConnectAsync(-111); });
             }
         }
+        [Fact]
```

Comment:
again since Pipe tests are a little heavyweight, and we know timeout vs int has zero connection with the functionality, can you reduce the number of new pipe tests or avoid any by just changing half of the existing timeouts in the tests to use TimeSpan instead?

Reply:
(as long as every overload gets called once somewhere)